x-pack/filebeat/docs/inputs/input-cel.asciidoc

```diff
@@ -122,7 +122,7 @@ The field should be an array, but in the case of an error condition in the CEL p
 
```

Comment:
Up on line 16 there's a type for `incled`.

Reply:
Fixed the really weird English that I chose to use too.